src/Glpi/Application/View/Extension/DataHelpersExtension.php

```diff
@@ -69,6 +70,13 @@ public function getFilters(): array
         ];
     }
 
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('array_diff', 'array_diff'),
+        ];
+    }
+
```

Comment:
IMHO, template logic should be simple enough to not have to use functions like `array_diff`. Indeed, if logic requires to do array manipulations, then it should probably be tested by a PHPUnit test, and therefore should be placed in a class method.